### PR TITLE
Use customer note as special instructions

### PR DESCRIPTION
**Pull request by WilRyan**

### CODE REVIEW

File: MdsSupportingClasses/MdsColliveryService.php [hunk 1]
```diff
@@ -584,9 +584,17 @@ public function automatedAddCollivery($order_id, $processing = false)
...
+                    if ($this->settings->getValue('include_customer_note') == 'yes') {
+                        $instructions .= $order->get_customer_note();
+                    }
+                    if ($this->settings->getValue('include_order_number') == 'yes') {
+                        if(strlen($instructions)>0)$instructions .= ' ';
+                        $instructions .= 'Order number: ' . $order_id;
+                    }
                     if ($this->settings->getValue('include_product_titles') == 'yes') {
                         $count = 1;
+                        if(strlen($instructions)>0)$instructions .= ' ';
```
Comment (MarnuLombard) [inline MdsSupportingClasses/MdsColliveryService.php]: Please modify this syntax into:
```
if () {
  $instrunctions .= ' ';
}
```

File: collivery.php [hunk 2]
```diff
@@ -3,14 +3,14 @@
 use MdsSupportingClasses\MdsColliveryService;
 
 define('_MDS_DIR_', __DIR__);
-define('MDS_VERSION', '3.3.3');
```
Comment (MarnuLombard) [inline collivery.php]: Should be a minor version bump, not a patch.
See https://semver.org

File: mds_admin.php [hunk 3]
```diff
@@ -549,14 +549,22 @@ function mds_register_collivery()
...
+    if ($this->settings->getValue('include_customer_note') == 'yes') {
+        $instructions .= $order->get_customer_note();
+    }
+    if ($this->settings->getValue('include_order_number') == 'yes') {
+        if(strlen($instructions)>0)$instructions .= ' ';
+        $instructions .= 'Order number: ' . $order_id;
+    }
+    if ($this->settings->getValue('include_product_titles') == 'yes') {
         $count = 1;
+        if(strlen($instructions)>0)$instructions .= ' ';
```
Comment (MarnuLombard) [inline mds_admin.php]: Please add full `if()` syntax

File: MdsSupportingClasses/MdsFields.php [hunk 4]
```diff
@@ -43,6 +43,18 @@ public static function getFields()
...
+            'include_order_number' => array(
+                'title' => __('Include order number'),
+                'type' => 'checkbox',
+                'description' => __('Includes order number in the delivery instructions, max 4096 characters'),
+                'default' => 'yes',
+            ),
+            'include_customer_note' => array(
+                'title' => __('Include customer note'),
+                'type' => 'checkbox',
+                'description' => __('Includes customer note in the delivery instructions, max 4096 characters'),
```
Comment (MarnuLombard) [inline MdsSupportingClasses/MdsFields.php]: Update text about 4096 being the total.

File: MdsSupportingClasses/MdsColliveryService.php (hunk 1 above)
Comment (WilRyan) [reply MdsSupportingClasses/MdsColliveryService.php]: 93c631c9188d5a558d1246baf7ea93d007562317

**Pull request merged**